src/routes.json

```diff
@@ -101,20 +101,25 @@
       "component": "pickupLabRequestAction",
       "slot": "tests-ordered-actions-slot"
     },
-    {
+   {
```

Comment:
One final thing before we get this in, this indentation change looks unintentional